.github/workflows/impress.yml

```diff
@@ -178,6 +178,9 @@ jobs:
             -v /data/media:/data \
             minio/minio server --console-address :9001 /data
 
+      - name: Add delay to wait for Minio to start correctly
+        run: sleep 5
```

Comment:
maybe use something like this to avoid wasting time?
https://github.com/openfun/joanie/blob/main/.circleci/config.yml#L227